iphone/Classes/TiViewProxy.m

```diff
@@ -1252,7 +1252,8 @@ -(void)_initWithProperties:(NSDictionary*)properties
 	// Set horizontal layout wrap:true as default 
 	layoutProperties.layoutFlags.horizontalWrap = YES;
 	[self initializeProperty:@"horizontalWrap" defaultValue:NUMBOOL(YES)];
-	
+	[self initializeProperty:@"visible" defaultValue:NUMBOOL(YES)];
```

Comment:
This is going to cause a problem with `Ti.UI.ActivityIndicator`. Setting visible to true by default will cause the activity indicator to appear as soon as it is added to the window hierarchy even if `show()` is never called.